antarest/study/web/variant_blueprint.py

```
@@ -82,8 +82,12 @@ def create_variant(uuid: str, name: str) -> str:
                             "antares": {
                                 "version": variant_study.version,
                                 "caption": variant_study.name,
-                                "created": variant_study.created_at.timestamp(),
-                                "lastsave": variant_study.created_at.timestamp(),
+                                "created": variant_study.created_at.timestamp()
+                                if variant_study.created_at
```

I think we should raise if those fields are not defined here. We have create the variant_study just above, they must be defined. The database schema is too lax, but we should be more rigorous if we know the application should ensure non-nulls.
We can add getters to the study class that return a non optional or raise